Main/src/intarfaceIWalker.kt

```diff
@@ -1,3 +1,4 @@
 interface IWalker{
+    val speed: Int
```

Comment:
If you want to declare variables, it's better to use an abstract class. In interfaces, it is not customary to declare variables